File: BlackJack.WebApi/Models/WinningRules.cs
namespace BlackJack.WebApi.Models
{
    public class WinningRules
    {
        public bool IsPlayerBlackJack { get; set; }
        public bool IsTie { get; set; }
        public bool IsDealerBlackJack { get; set; }
        public bool IsDealerBusts { get; set; }
        public bool IsPlayerBusts { get; set; }
        public bool IsPlayerCloseToBlackJack { get; set; }
        public bool IsDealerCloseToBlackJack { get; set; }
    }
}

File: BlackJack.TypeScript/src/models/WinningRules.ts
import { Model } from "sequelize/types";


export class WinningRules {
    isPlayerBlackJack: any;
    isTie: any;
    isDealerBlackJack: any;
    isDealerBusts: any;
    isPlayerBusts: any;
    isPlayerCloseToBlackJack: any;
    isDealerCloseToBlackJack: any;
    isPlayerHasAceAndPontoon: any;
    isPlayerHasFiveTurn: any;
   

    constructor(isPlayerBlackJack ?: any, isTie ?: any, isDealerBlackJack ?: any, isDealerBusts ?: any
          ,isPlayerBusts ?: any, isPlayerCloseToBlackJack ?: any, isDealerCloseToBlackJack?: any, isPlayerHasAceAndPontoon?: any 
          , isPlayerHasFiveTurn?: any
          )
        { this.isPlayerBlackJack = isPlayerBlackJack;
        this.isTie = isTie;
        this.isDealerBlackJack = isDealerBlackJack;
        this.isDealerBusts = isDealerBusts;
        this.isPlayerBusts = isPlayerBusts;
        this.isPlayerCloseToBlackJack = isPlayerCloseToBlackJack,
        this.isDealerCloseToBlackJack = isDealerBlackJack,
        this.isPlayerHasAceAndPontoon = isPlayerHasAceAndPontoon;
        this.isPlayerHasFiveTurn = isPlayerHasFiveTurn

}

}